components/agentSearchForm/index.jsx

```diff
@@ -57,6 +64,13 @@ function AgentSearchForm({
       {invalidWebIdField ? (
         <Message variant="invalid">Please provide a valid WebID</Message>
       ) : null}
+      {existingWebId ? (
+        <Message variant="error">
+          {session.info.webId === existingWebId
+            ? "You cannot overwrite your own permissions."
+            : `The WebID ${existingWebId} is already in your permissions.`}
+        </Message>
+      ) : null}
       <SimpleInput
```

Comment:
You also want the input field to show being invalid - use the modifier `invalid` to do that (or with the React component, use the property `variant` with `invalid`).